pyproject.toml

```diff
@@ -48,6 +48,7 @@ _ = "returns.contrib.hypothesis._entrypoint"
 python = "^3.7"
 
 typing-extensions = ">=4.0,<5.0"
+mypy = { version = "^1.2", optional = true }
```

Comment:
I think we can now remove `mypy` from `dev-dependencies`

Reply:
Why do you think so? In this case, `poetry install` won't install mypy.

Reply:
I was checking and if we remove the mypy from the dev-dependencies, then in order to let the python installed, we need to do:
```bash
$ poetry --only main --extras compatible-mypy
```

Reply:
Some small example https://asciinema.org/a/N6NpAQsKGc6T4xR6DOfDQFxDn

Reply:
https://asciinema.org/a/VOKGDmAFQBA8CgaZkleKvVQs8